### DISCUSSION TITLE
zh/plugins/blog/comment/guide

### DISCUSSION THREAD

**Discussion by giscus** [vuepress/ecosystem#216] [Announcements]
# zh/plugins/blog/comment/guide

指南 设置选项 你既可以在 Node.js 一侧使用插件选项设置选项，也可以通过客户端配置文件在浏览器一侧设置选项。 通过插件选项 通过客户端配置文件 有以下你需要注意的限制： provider、多语言设置和其他资源相关选项必须在插件选项中设置。 为确保 tree-shaking 有效，我们必须在 Node 一侧优化入口，以便打包器可以了解最终打包中应...

https://ecosystem.vuejs.press/zh/plugins/blog/comment/guide.html

**Comment by zhu8347105** [comment]
现在国内有个支持vuepress三方评论系统，叫睿言，界面和交互还可以
官网：http://www.ruispeak.cn/ 大家有兴趣可以看看
